python-backend/wolfram_client.py

```diff
@@ -29,4 +36,18 @@ def ask(query: str) -> str:
 
     @staticmethod
     def limit(fraction: str) -> str:
-        return WolframClient.ask(query="limit of {}".format(fraction))
+        try:
+            result = WolframClient.ask(query="limit of {}".format(fraction), include_pod="Limit")
+            # only want to return: queryresult -> pods[0] -> subpods
+            return result["queryresult"]["pods"][0]["subpods"]
+        except Exception as e:
+            print("Failed to parse Wolfram API result", e)
```

Comment:
Should this use `logging` rather than `print`?

Reply:
great catch!